sound/soc/sof/amd/Kconfig

```diff
@@ -23,6 +23,7 @@ config SND_SOC_SOF_AMD_COMMON
 	select SND_AMD_ACP_CONFIG
 	select SND_SOC_SOF_XTENSA
 	select SND_SOC_SOF_ACP_PROBES
+	select SND_SOC_ACPI_AMD_MATCH
```

Comment:
this is weird, you have multiple selections of SND_SOC_ACPI.

config SND_SOC_ACPI_AMD_MATCH
	tristate
	select SND_SOC_ACPI if ACPI

Reply:
its similar to Intel Kconfig option 'config 'SND_SOC_ACPI_INTEL_MATCH'.

Reply:
ok, I don't fully remember why we did this, but if it ain't broke don't fix it...